cfgov/housing_counselor/jinja2/housing_counselor/improved_index.html

```diff
@@ -31,17 +31,39 @@ <h1>Find a housing counselor</h1>
                             <div class="o-featured-content-module_text">
                                 <form class="o-form" action="#">
                                     <div class="m-form-field-with-button">
+
                                         <div class="m-form-field">
                                             <label class="a-label a-label__heading" for="hud_hca_api_query">
                                                 Search by ZIP code:
                                             </label>
+
+                                            {% if zipcode and not zipcode_valid %}
+                                            <div class="u-mb15">
+                                                <div class="m-notification
+                                                            m-notification__default
+                                                            m-notification__error
+                                                            m-notification__visible">
+                                                    {{ svg_icon( 'warning-round' ) }}
+                                                    <div class="m-notification_content" role="alert">
+                                                        <div class="h4 m-notification_message">
+                                                            You have entered an invalid ZIP code.
+                                                        </div>
+                                                        <div class="m-notification_explanation">
+                                                            Enter a valid five-digit ZIP code.
```

Comment:
This bit of error text in the current version of the tool reads "Please enter a valid five-digit ZIP code above." Getting rid of the "above" in the new version makes sense since the form is no longer above the error, but should we leave the "Please" in there?

Reply:
See note in other comment. I vote for brevity, but you have the final call on this one.

Reply:
Fixed!